native/cocos/core/Root.cpp

```diff
@@ -371,8 +371,8 @@ void Root::frameMoveBegin() {
     _cameraList.clear();
 }
 
-void Root::frameMoveProcess(bool isNeedUpdateScene, int32_t totalFrames, const ccstd::vector<IntrusivePtr<scene::RenderWindow>> &windows) {
-    for (const auto &window : windows) {
+void Root::frameMoveProcess(bool isNeedUpdateScene, int32_t totalFrames) {
```

Comment:
Why change it, it is used for mutilple windows.

Reply:
The windows is member variable.It is redundant code and does not need to be passed through formal parameters

Reply:
Yep, you are right.